maflib/scheme_factory.py

```diff
@@ -109,21 +124,22 @@ def build_scheme_class(datum, base_scheme):
         column_desc = OrderedDict()
 
     # now create the scheme
+    # FIXME: Do not duck type MafScheme here
```

Comment:
`MafScheme` is an ABC, presumably different schemes would've been in Python instead of JSON.

Investigate updating the base class so this following section is no longer necessary.
e.g. factory method to programmatically create child classes.